docs/source/tutorials/token_classification.ipynb

```diff
@@ -277,6 +288,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
     "given_words, given_labels = [], [] \n",
     "\n",
     "for filepath in filepaths: \n",
```

Comment:
For hidden cell at the very bottom (which throws the exception and serves as another "unit test"), let's add comment at the top:

\# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.